README.md

```diff
@@ -83,6 +83,13 @@ There are a few important considerations to be aware of:
 
 5. If your Pages site is using GitHub Actions as the source, while not required we highly recommend you also [protect your environment][environment-protection] (we will configure it by default for you).
 
+### OICD
```

Comment:
```suggestion
### OIDC
```

Reply:
this applies to all the place refer to that :)

Reply:
🙈